files/en-us/learn/server-side/express_nodejs/deployment/index.html

```diff
@@ -322,17 +332,19 @@ <h3 id="Creating_an_application_repository_in_Github">Creating an application re
 </ol>
 
 <div class="notecard warning">
-<p><strong>Warning:</strong> In 2020 Github changed the default repo branch name to "main" (from "master"). If using an older/existing repository you might need to call <code>git push origin master</code> instead.</p>
+  <h4>Warning</h4>
+  <p>In 2020 Github changed the default repo branch name to "main" (from "master"). If using an older/existing repository you might need to call <code>git push origin master</code> instead.</p>
```

Comment:
I believe, they stylise their product as
```suggestion
  <p>In 2020 GitHub changed the default repo branch name to "main" (from "master"). If using an older/existing repository you might need to call <code>git push origin master</code> instead.</p>
```